GameData/ETT/EngTechTree.cfg

```diff
@@ -912,6 +906,8 @@ TechTree
 			part = a_6m_cargo
 			part = a_cockpit_adaptor
 			part = B9_Cockpit_MK5
+			part = Airlock

```

Comment:
I believe it to be a command module. Intended for EVAing kerbals in the cargo bay